package.json

```diff
@@ -81,7 +82,7 @@
     "redux-thunk": "2.3.0",
     "reselect": "4.0.0",
     "use-query-params": "1.1.9",
-    "vanilla-framework": "2.19.3"
+    "vanilla-framework": "2.19.2"
```

Comment:
Why roll Vanilla back?

Reply:
To match with the version currently on react-components.

Reply:
I've bumped this back again.